cas-airflow/values.yaml

```diff
@@ -2,7 +2,13 @@ airflow:
   airflow:
     image:
       repository: docker-registry.default.svc:5000/wksv3k-tools/cas-airflow
-      tag: 0.0.1
+      tag: '0.1.0-pre.022'
+    config:
+      AIRFLOW__CORE__REMOTE_LOGGING: True
+      AIRFLOW__ELASTICSEARCH__HOST: 'airflow'
+      AIRFLOW__ELASTICSEARCH__LOG_ID_TEMPLATE: '{{dag_id}}-{{task_id}}-{{execution_date}}-{{try_number}}'
+      AIRFLOW__ELASTICSEARCH__END_OF_LOG_MARK: 'end_of_log'
+      AIRFLOW__ELASTICSEARCH__WRITE_STDOUT: 'true'
```

Comment:
`C_FORCE_ROOT` should be moved out of `extraEnv` too, right?

Reply:
@wenzowski I don't think so? It sounds to me like C_FORCE_ROOT needs to be an env variable in order to run celery as root, whereas variables like AIRFLOW_ELASTICSEARCH_HOST are being inserted into the ariflow.cfg, which is why they are under airflow:config:

Reply:
https://stackoverflow.com/questions/20346851/running-celery-as-root

Reply:
Done mucking; though now wondering if this is the right approach. By writing logs to a fake elasticsearch we can't read them back. Maybe we should be writing them to an object storage bucket instead?